### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by gawi151**
Add issue templates for easier contribution to the project and a more structured way the issues are created.

templates cover:
- bug report
- feature request

created based on [comment](https://github.com/nomtek/mistralai_client_dart/pull/24#discussion_r1457365413)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior
```
Comment (KacperKluka) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Let's be consistent 😁 
```suggestion
Steps to reproduce the behavior.
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
```
Comment (KacperKluka) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Shouldn't it be the "ENHANCEMENT" label? 🤔

**Review by KacperKluka — CHANGES_REQUESTED** [review]
Love it ❤️ Just a few small things 😉

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (gawi151) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Updated in https://github.com/nomtek/mistralai_client_dart/pull/25/commits/8cd768d5fd5da1da3dfa28e4f126b264a99808cb

**Review by KacperKluka — APPROVED** [review]
LGTM 👍

**Pull request merged**